core/pkg/runtime/from_config.go

```diff
@@ -60,7 +60,7 @@ type SourceConfig struct {
 // Config is the configuration structure derived from startup arguments.
 type Config struct {
 	MetricExporter    string
-	MetricsPort       uint16
+	ManagementPort    uint16
```

Comment:
You also need to change where this configuration is being used.

For example, this configuration is propagated to the service level. See this usage - https://github.com/open-feature/flagd/blob/main/core/pkg/runtime/from_config.go#L137 . And then you can change the name of that usage too.